README.md

```diff
@@ -122,6 +122,14 @@ Celerity's runtime behavior:
   at the end of execution (requires log level `info` or higher).
 - `CELERITY_DRY_RUN_NODES` takes a number and simulates a run with that many nodes
   without actually executing the commands.
+<<<<<<< HEAD
```

Comment:
I think you might have committed a merge conflict?

Reply:
Oops!